Device_Profile_Sample/Device_Profile_Get_Identity_Data/Program.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
```

Comment:
This is just a copy of the sample from ADAL - I want it to be specific to VSTS, aka call a REST API or client library call like the other samples.  Remove the cruft like the colors, comments, command line, etc. and make this match the other samples